examples/hybrid/tuning/mwe_tune_ke.jl

```diff
@@ -0,0 +1,163 @@
+    h_elem = 12
+    println(
+        "running KE tuning test on $(context.device); h_elem = $h_elem; z_elem = $z_elem; npoly = $npoly; R = $R; z_max = $z_max; FT = $FT",
+    )
+    # horizontal space
+    domain = Domains.SphereDomain(R)
+    horizontal_mesh = Meshes.EquiangularCubedSphere(domain, h_elem)
+    horizontal_topology = Topologies.Topology2D(
+        context,
+        horizontal_mesh,
+        Topologies.spacefillingcurve(horizontal_mesh),
+    )
+    horizontal_topology_cpu = Topologies.Topology2D(
+        context_cpu,
+        horizontal_mesh,
+        Topologies.spacefillingcurve(horizontal_mesh),
+    )
+    quad = Spaces.Quadratures.GLL{npoly + 1}()
+    h_space = Spaces.SpectralElementSpace2D(horizontal_topology, quad)
+    h_space_cpu = Spaces.SpectralElementSpace2D(horizontal_topology_cpu, quad)
+
+    # vertical space
+    z_domain = Domains.IntervalDomain(
+        Geometry.ZPoint(zero(z_max)),
+        Geometry.ZPoint(z_max);
+        boundary_tags = (:bottom, :top),
+    )
+    z_mesh = Meshes.IntervalMesh(z_domain, nelems = z_elem)
+    z_topology = Topologies.IntervalTopology(context, z_mesh)
+    z_topology_cpu = Topologies.IntervalTopology(context_cpu, z_mesh)
+
+    z_center_space = Spaces.CenterFiniteDifferenceSpace(z_topology)
+    z_center_space_cpu = Spaces.CenterFiniteDifferenceSpace(z_topology_cpu)
+
+    z_face_space = Spaces.FaceFiniteDifferenceSpace(z_topology)
+    z_face_space_cpu = Spaces.FaceFiniteDifferenceSpace(z_topology_cpu)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(h_space, z_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+
+    hv_center_space_cpu =
+        Spaces.ExtrudedFiniteDifferenceSpace(h_space_cpu, z_center_space_cpu)
+    hv_face_space_cpu =
+        Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space_cpu)
+
+    # CPU
+    ᶜlocal_geometry_cpu = Fields.local_geometry_field(hv_center_space_cpu)
+    ᶠlocal_geometry_cpu = Fields.local_geometry_field(hv_face_space_cpu)
+    uₕ_cpu = center_initial_condition(ᶜlocal_geometry_cpu, R)
+    uᵥ_cpu = face_initial_condition(ᶠlocal_geometry_cpu)
+    κ_cpu = init_scalar_field(hv_center_space_cpu)
+
+    # GPU
+    ᶜlocal_geometry = Fields.local_geometry_field(hv_center_space)
+    ᶠlocal_geometry = Fields.local_geometry_field(hv_face_space)
+    uₕ = center_initial_condition(ᶜlocal_geometry, R)
+    uᵥ = face_initial_condition(ᶠlocal_geometry)
+    κ = init_scalar_field(hv_center_space)
+
+    return (; κ = κ, uₕ = uₕ, uᵥ = uᵥ, κ_cpu, uₕ_cpu = uₕ_cpu, uᵥ_cpu = uᵥ_cpu)
+end
+
+function profile_compute_kinetic()
+    κ, uₕ, uᵥ, κ_cpu, uₕ_cpu, uᵥ_cpu = initialize_mwe()
+    # compute kinetic energy
+    κ = compute_kinetic_ca!(κ, uₕ, uᵥ)
+    κ_cpu = compute_kinetic_ca!(κ_cpu, uₕ_cpu, uᵥ_cpu)
+
+    @show Array(parent(κ)) ≈ parent(κ_cpu)
+
+    nreps = 10
+
+    for i in 1:nreps
+        NVTX.@range "($i). compute_kinetic_ca!" color = colorant"blue" begin
```

Comment:
It's a minor comment, but if you do
```suggestion
        NVTX.@range "compute_kinetic_ca!" color = colorant"blue" payload=i begin
```
you can avoid passing new strings each time